lib/minilib/unfmt.c

```diff
@@ -266,7 +266,7 @@ int main() {
 	
 	rv = unsfmt("hmm...?\t\t\nab0:cD:2345% \t\r934x5", "hmm...? %2x0:%2x:%d%% %2d", convs, convs+1, convs+2, convs+3);
 	
-	printf("%d: %x %x %d %d", rv, convs[0], convs[1], convs[2], convs[3]);
```

Comment:
This one we can leave -- it's part of the test driver and doesn't show up in RebbleOS

Reply:
Reverted.